src/common/legacy/src/graph_transformer.cpp

```diff
@@ -474,6 +474,22 @@ void ConstTransformer::fullTrim() {
     std::lock_guard<std::mutex> lock(lockFullTrim);
     auto sortedLayers = details::CNNSubnetSortTopologically({inputs, outputs});
     auto constMapLayers = getConstLayers(sortedLayers);
+
+    for (auto& layer : sortedLayers) {
```

Comment:
nit: 
```suggestion
    for (const auto& layer : sortedLayers) {
```
And the same for next cycles, because we don't change variables.